src/redis-benchmark.c

```diff
@@ -616,6 +647,44 @@ static void writeHandler(aeEventLoop *el, int fd, void *privdata, int mask) {
         if (config.cluster_mode && c->staglen > 0) setClusterKeyHashTag(c);
         atomicGet(config.slots_last_update, c->slots_last_update);
         c->start = ustime();
+        if (config.qps) {
```

Comment:
can we encapsulate this qps specific writeHandler logic in a dedicated function (eg. operateQPS or something) right now it makes the writehandler very complicated in appearance